ometiff-JavaScript/PixelSizeConsistency.js

```diff
@@ -1,37 +1,41 @@
 const ExifReader = require('exifreader')
 const parseString = require('xml2js').parseString
 
-const validunits = ['um', 'nm', 'mm']
+const validUnitsInOME = ['µm', 'nm', 'mm']
+const validUnitsInJSON = ['um', 'nm', 'mm']
 
 function getExif() {
     const tags = ExifReader.load("test_image.ome.tif")
     return tags
 }
 
-function convertFactor(fromUnit, toUnit) {
-    if(!validunits.includes(fromUnit) || !validunits.includes(toUnit)){
-        console.log('\x1b[31mWARNING: PixelSize consistency is only validated for "mm", "um" and "nm". PhysicalUnit in OME-TIFF is "%s", consistency check is skipped.\x1b[0m', fromUnit)
+function convertFactor(omeUnit, jsonUnit) {
+    if(!validUnitsInOME.includes(omeUnit)){
+        console.log('\x1b[31mERROR: PixelSize consistency is only validated for "mm", "µm" and "nm". PixelSizeUnits in OME-TIFF is "%s", consistency check is skipped.\x1b[0m', omeUnit)
```

Comment:
```suggestion
        console.log('\x1b[31mERROR: PixelSize consistency is only validated for "mm", "µm" and "nm". PhysicalUnit in OME-TIFF is "%s", consistency check is skipped.\x1b[0m', omeUnit)
```

Hey, sorry that was not clear. Let's keep "PhysicalUnit" for OME-TIFF and "PixelSizeUnits" for JSON i.e. refering to the real name of the field for each of them.

Reply:
Done